src/main/java/com/michelin/kafkactl/ConnectClustersSubcommand.java

```diff
@@ -0,0 +1,140 @@
+package com.michelin.kafkactl;
+
+import com.michelin.kafkactl.models.Resource;
+import com.michelin.kafkactl.services.FormatService;
+import com.michelin.kafkactl.services.LoginService;
+import com.michelin.kafkactl.services.ResourceService;
+import jakarta.inject.Inject;
+import lombok.Getter;
+import picocli.CommandLine;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ * Represents the Kafka Connect Cluster sub command class.
+ */
+@CommandLine.Command(name = "connect-clusters", description = "Interact with connect clusters (Vaults)")
+public class ConnectClustersSubcommand implements Callable<Integer> {
+
+    /**
+     * Gets or sets the kafkactl parent command.
+     */
+    @CommandLine.ParentCommand
+    public KafkactlCommand kafkactlCommand;
+
+    /**
+     * Gets or sets the command specification.
+     */
+    @CommandLine.Spec
+    public CommandLine.Model.CommandSpec commandSpec;
+
+    /**
+     * Gets or sets the sub command action.
+     */
+    @CommandLine.Parameters(index = "0", description = "vaults", arity = "1")
```

Comment:
You can use: 

`@CommandLine.Parameters(index = "0", description = "Action to perform (${COMPLETION-CANDIDATES}).", arity = "1")`

To automatically list all the available actions